### PR TITLE
Image priority

### PR DESCRIPTION
**Pull request by apledger**

### CODE REVIEW

**Review by migueloller — COMMENTED** [review]
Changes to the Image component look good, but I'm not sure I understand how the changes for background image would work.

Where's the priority coming from?

File: packages/runtime/src/prop-controllers/descriptors.ts [hunk 1]
```diff
@@ -205,6 +205,7 @@ type BackgroundImageV0 = {
   repeat?: BackgroundImageRepeat
   opacity?: number
   parallax?: number
+  priority?: boolean
```
Comment (migueloller) [inline packages/runtime/src/prop-controllers/descriptors.ts:208]: Where would this value be coming from? Are we also updating the backgrounds panel to add a priority?

File: packages/runtime/src/prop-controllers/descriptors.ts [hunk 2]
```diff
@@ -215,6 +216,7 @@ type BackgroundImageV1 = {
...
```
Comment (migueloller) [inline packages/runtime/src/prop-controllers/descriptors.ts:219]: See my other comment on adding `priority` to `BackgroundImageV0`

**Review by migueloller — CHANGES_REQUESTED** [review]
Also, don't forget to add a changeset @apledger

File: packages/runtime/src/prop-controllers/descriptors.ts (hunk 1 above)
Comment (apledger) [reply packages/runtime/src/prop-controllers/descriptors.ts:208]: Yes. I figured we'd merge this first to expect the priority panel before we add it to the UI.

File: packages/runtime/src/prop-controllers/descriptors.ts (hunk 2 above)
Comment (apledger) [reply packages/runtime/src/prop-controllers/descriptors.ts:219]: Same response.

**Review by migueloller — APPROVED** [review]

**Pull request merged**